README.md

```diff
@@ -1,26 +1,78 @@
+<!-- markdownlint-disable MD026 MD033 -->
+
 # DiscordFeedback
-The bot that powers the Discord Feedback server
 
-# Selfhosting
-We don't (fully) support hosting your own copy of DiscordFeedback, this program is highly customized and not suited to run outside of the official Discord Feedback server.
+This is the bot that powers the Discord Feedback server, develop'd by the volunteer group only known as "Discord Feedback Bot Tinkerers Sqaud™"
```

Comment:
I really don't like this sentence, can we be a tad more serious?

Reply:
I agree with that, will change.

Reply:
Also "develop**ed**".